pom.xml

```diff
@@ -11,14 +11,14 @@
     <packaging>jar</packaging>
     <groupId>org.webjars</groupId>
     <artifactId>angularjs</artifactId>
-    <version>1.3.17-2-SNAPSHOT</version>
+    <version>1.4.4-1-SNAPSHOT</version>
```

Comment:
Typically we don't amend the `-1` until we need a new version of the upstream.  So this should be `1.4.4-SNAPSHOT`.  Sorry for the hassle.